app/models/conversion/project.rb

```diff
@@ -9,4 +9,10 @@ def route
     return :sponsored if directive_academy_order?
     :voluntary
   end
+
+  def fetch_provisional_covnversion_date
```

Comment:
Typo in the method name, should be `fetch_provisional_conversion_date`

Reply:
I forgot to even use this! I've soted it now, it is used in the stakeholder kick off task view.